<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="homeButton">
       <property name="text">
        <string>H</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/images/noun_148012_cc.png</normaloff>:/images/noun_148012_cc.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="backwardButton">
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/images/noun_148014_cc.png</normaloff>:/images/noun_148014_cc.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton">
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/images/noun_148032_cc.png</normaloff>:/images/noun_148032_cc.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="HelpBrowser" name="helpBrowser"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpBrowser</class>
   <extends>QTextBrowser</extends>
   <header>help/helpbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
